### ISSUE TITLE
Sample of how to use it within live wallpaper?

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [googlesamples/android-vulkan-tutorials#49]
Maybe show an image ?
Or a video?

**Comment by AndroidDeveloperLB** [comment]
I don't understand.
Is there a sample of how to use it in live wallpaper?

**Comment by ggfan** [comment]
not from ndk samples. But AR samples do that ( I believe )

**Comment by AndroidDeveloperLB** [comment]
There are AR samples?!
Where?
I wanna try them !

**Comment by AndroidDeveloperLB** [comment]
And actually, how could an AR work on live wallpaper? Show the camera there?

**Comment by ggfan** [comment]
yeah, they use camera as texture, then add 3D graphics characters ( so Camera is "Reality", 3D graphics is "augmented" part)

**Comment by AndroidDeveloperLB** [comment]
How cool.
A bit weird for live wallpaper, but still cool.

**Comment by ggfan** [comment]
I might understand you wrong, though. Anyway, try browse this:  https://developers.google.com/ar/develop/c/quickstart  to see whether it could be useful.

**Comment by ggfan** [comment]
Oh, AR samples categorized as Java, C, and Unity, once in that page, you could explore more. good luck!

**Comment by Andreyogld3d** [comment]
How to use Vullkan for live wallpaper without Java ?

**Comment by AndroidDeveloperLB** [comment]
@ggfan I don't think this website has live wallpaper sample.

**Comment by Andreyogld3d** [comment]
@ggfan We need an information how to create a live wallpaper using Vullkan api

@AndroidDeveloperLB 

> I don't think this website has live wallpaper sample.

Anyway we need it!

**Comment by AndroidDeveloperLB** [comment]
@Andreyogld3d I know. My request was written a long time ago that I even forgot about it.
